src/frontend/components/sidebar/sidebar.css

```diff
@@ -0,0 +1,53 @@
+
+.sidebar-container{
+    margin-top: 0.938rem;
+    display: flex;
+    flex-direction: column; 
+    align-items: flex-start;
+    
+}
+.sidebar-link{
+    display: flex;
+    gap: 2rem;
+    min-width: 12.5rem;
+    padding: 1.125rem 1.313rem;
+    align-items: center;
+    border-radius: 0rem 0.938rem 0.938rem 0rem;
```

Comment:
Why you are using such random numbers here, Is there any specific reason ...?

Reply:
no i just converted these from px to rem